Automattic/jetpack
# Connection: Userless button to WP.com

#### Changes proposed in this Pull Request:
The "Userless" button is getting moved to WP.com, and this PR accompanies the change.

In userless mode we:
- increase size of the iframe
- remove the button container, JS and styling that accompany it

#### Jetpack product discussion
p9dueE-2z0-p2#comment-4591

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
0. Apply the diff D58840-code and switch to sandbox.
1. Disconnect Jetpack, disable Userless mode if enabled.
2. Connect Jetpack using In-Place flow, confirm that you get connected, check for styling issues.
3. Log in as a secondary user, connect using in-place flow. Confirm that you get connected, check for styling issues.
4. Log out of WP.com, disconnect Jetpack, enable Userless mode (e.g. `define( 'JETPACK_NO_USER_TEST_MODE', true );`).
5. Go to "Jetpack" section, click "Set up Jetpack". Confirm that the iframe looks like this: <img width="583" alt="Screen Shot 2021-03-16 at 3 35 09 PM" src="https://user-images.githubusercontent.com/1341249/111376257-3b564980-866d-11eb-9573-582f1b8101a3.png">
6. Click "Continue without signing in". Confirm that you're redirected to the Jetpack Cloud pricing page: `https://cloud.jetpack.com/pricing/...`.
7. Click "Start for Free", then go to "Jetpack -> Settings".
8. Find a disabled module card, click "Connect my WordPress.com account". Confirm that the connection card underneath looks like this: <img width="1056" alt="Screen Shot 2021-03-16 at 3 39 18 PM" src="https://user-images.githubusercontent.com/1341249/111376698-cb948e80-866d-11eb-9458-c773e745dcfc.png">
9. Complete the user connection, confirm that you're fully connected to Jetpack.
10. At this point you should be logged into WP.com. Disconnect Jetpack, and initiate the connection process again.
11. Confirm that you're not offered the userless mode, because you're logged into WP.com.
